dpm/client/__init__.py

```diff
@@ -13,10 +13,6 @@
 class DpmException(Exception):
```

Comment:
@Fak3 one small comment i'm making here: can we get more descriptive commits e.g. here you also moved config around. Also more smaller commits would be easier to review.